### DISCUSSION TITLE
What is your favorite terminal????

### DISCUSSION THREAD

**Discussion by justinzw** [Seagate/cortx#378] [General] [answered]
At the risk of starting a debate. I'd like to ask everyone what their favorite terminal is?

I used to use https://www.iterm2.com/ on Mac but most recently switched to Windows. 

I started using this - https://github.com/microsoft/terminal and it's quite cool!

Anyone else have any cool suggestions? alternatives?

**Comment by yanqingfu** [comment, marked as answer]
[windows MobaXterm](https://mobaxterm.mobatek.net/)

**Comment by johnbent** [comment]
@justinzw what do you want to use it for?  I want something that gives me bash, gives me Unix utilities, gives me python, gives me some sort of installation tool like yum.  Windows Subsystem Linux is actually really good.  Yeah, I loved iTerm back in my Mac days also.  :)

**Reply by justinzw** [reply]
Thanks @yanqingfu  I started using MobaXterm and it looks really cool, I'm going to continue to use it. It seems to have a ton of features, including "games" :-) 

@johnbent I mostly want to use it for running things in the command line and it will be nice to have UI features like tabs and remembering session settings etc... I'll also try Windows Subsystem Linux. Thanks for the suggestion.

**Reply by yanqingfu** [reply]
yes, you can also ssh to remote server

**Reply by rahmed-stx** [reply]
@johnbent, Is MobaXterm allowed to be installed in seagate laptops, that we use at US campuses? I don't see it in the windows software center.

**Reply by yanqingfu** [reply]
@rahmed-stx from IT, you can chat with IT too

> Mobaxterm required license
> Please purchase the professional version of mobaxterm. That is the version approve to use in Seagate.
>  
> Here are the steps:
>  
> Check for the software in [iProcurement](https://seagatetechnology.sharepoint.com/sites/GMSites/factory-lab-security-and-asset-mgmt/SitePages/Purchasing-Licenses-via-iProcurement.aspx)  [how to login https://seagate.service-now.com/nav_to.do?uri=%2Fkb_view.do%3Fsysparm_article%3DKBA0011099]
>  
> a. If the Software is in iProcurement, proceed with the purchase
>  
> b. If the Software is not in iProcurement:
>  
> i. Request a quote for the product from SHI, using one of the following 3 emails depending upon the location:
>  
> - SeagateTechAsia@shi.com (for Asia users)
>  
> - SeagateTechEurope@shi.com (for Europe users)
>  
> - SeagateTechUS@shi.com (for US users)
>  
> Once you have the quote, submit a non-catalog purchase requisition (PR) request through iProcurement and attach the quote.

**Comment by mukul-seagate11** [comment]
I would say MobaXterm for windows but on the other hand I would say online Interactive terminal, https://www.katacoda.com are more cool If we can deliver our product so that everyone can get an opportunity to explore & test the CORTX software on the free terminal which is available to all community
